tests/integration/kw_version_test.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+
+include './src/lib/kwio.sh'
+include './tests/unit/utils.sh'
+include './tests/integration/utils.sh'
+
+function oneTimeSetUp()
+{
+  setup_container_environment
+}
+
+function kw_version_test_helper()
+{
+  local distro="$1"
+  local container
+  local kw_git_dir
+  local kw_dir
+  local head_hash
+  local branch_name
+  local base_version
+  local expected_output
+  local output
+
+  kw_dir="${KWROOT_DIR}"
+  kw_git_dir="${kw_dir}/.git"
+
+  # In order to check correctness of `kw --version`, we collect some information
+  # from the git repo:
+  #
+  # - Base version (alpha, beta, or other)
+  # - Branch name
+  # - Commit sha
+  #
+  # Because the local KW repo is copied to the container, we run  the  following
+  # commands directly on the host instead of running in the container.
+  head_hash=$(git --git-dir "${kw_git_dir}" rev-parse --short HEAD)
+  branch_name=$(git --git-dir "${kw_git_dir}" rev-parse --short --abbrev-ref HEAD)
+  base_version=$(cat "${kw_dir}/src/VERSION" | head -n 1)
```

Comment:
Check:
* https://kworkflow.org/content/codingstyle.html#command-substitution-and-arithmetic-expression
* https://kworkflow.org/content/codingstyle.html#avoid-short-options